lib/src/abstraction_layer/policy/base/canvas_widgets_policy.dart

```diff
@@ -20,4 +20,7 @@ mixin CanvasWidgetsPolicy on BasePolicySet {
   List<Widget> showCustomWidgetsOnCanvasForeground(BuildContext context) {
     return [];
   }
+
+  /// Allows you to change whether to display links in front of components or vice versa.
+  bool displayLinksInFrontOfComponents() {return true;}
```

Comment:
This line is not formatted.

Can you change it to a getter like below, please?

`bool get showLinksOnTopOfComponents => true;`